test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua

```diff
@@ -0,0 +1,1045 @@
+    end)
+  m.getMobileSession(pAppId):ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  m.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      m.hashId[pAppId] = data.payload.hashID
+    end)
+end
+
+--[[ @createIntrerationChoiceSet: adding createIntrerationChoiceSet
+--! @parameters:
+--! pAppId - application number (1, 2, etc.)
+--! @return: none
+--]]
+function m.createIntrerationChoiceSet(pAppId)
+  if not pAppId then pAppId = 1 end
+  local choice = {
+    choiceID = pAppId,
+    menuName = "Choice" .. m.getConfigAppParams(pAppId).appName,
+    vrCommands = { "VrChoice" ..m.getConfigAppParams(pAppId).appName }
+  }
+  local cid = m.getMobileSession(pAppId):SendRPC("CreateInteractionChoiceSet", {
+      interactionChoiceSetID = pAppId,
+      choiceSet = { choice }
+    })
+  m.getHMIConnection():ExpectRequest("VR.AddCommand")
+  :Do(function(_, data)
+      m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+      m.resumptionData[pAppId].createIntrerationChoiceSet = { VR = data.params }
+    end)
+  m.getMobileSession(pAppId):ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  m.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      m.hashId[pAppId] = data.payload.hashID
+    end)
+end
+
+--[[ @setGlobalProperties: adding setGlobalProperties
+--! @parameters:
+--! pAppId - application number (1, 2, etc.)
+--! @return: none
+--]]
+function m.setGlobalProperties(pAppId)
+  if not pAppId then pAppId = 1 end
+  local params = {
+    helpPrompt = {
+      {
+        text = "Help prompt" .. pAppId,
+        type = "TEXT"
+      }
+    },
+    timeoutPrompt = {
+      {
+        text = "Timeout prompt" .. pAppId,
+        type = "TEXT"
+      }
+    },
+    vrHelpTitle = "VR help title" .. pAppId,
+    vrHelp = {
+      {
+        position = 1,
+        text = "VR help item" .. pAppId
+      }
+    },
+    menuTitle = "Menu Title" .. pAppId,
+  }
+  local cid = m.getMobileSession(pAppId):SendRPC("SetGlobalProperties", params)
+  m.resumptionData[pAppId].setGlobalProperties = {}
+  EXPECT_HMICALL("UI.SetGlobalProperties")
+  :Do(function(_,data)
+      m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+      m.resumptionData[pAppId].setGlobalProperties.UI = data.params
+    end)
+
+  EXPECT_HMICALL("TTS.SetGlobalProperties")
+  :Do(function(_,data)
+      m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+      m.resumptionData[pAppId].setGlobalProperties.TTS = data.params
+    end)
+
+  m.getMobileSession(pAppId):ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+  m.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_, data)
+      m.hashId[pAppId] = data.payload.hashID
+    end)
+end
+
+--[[ @subscribeVehicleData: adding subscribeVehicleData
+--! @parameters:
+--! pAppId - application number (1, 2, etc.)
+--! @return: none
```

Comment:
@GetmanetsIrina Please add description for `pParams` and `pHMIrequest` parameters